engine/src/flutter/shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -781,6 +781,18 @@ - (void)accessibilityElementDidLoseFocus {
   }
 }
 
+- (BOOL)accessibilityRespondsToUserInteraction {
+  if ((self.node.actions & ~flutter::kSystemActions) != 0) {
+    return true;
+  }
+
+  if (!self.node.customAccessibilityActions.empty()) {
```

Comment:
what's the diff between `self.node.actions` vs `self.node.customAccessibilityActions`

Reply:
iirc `customAccessibilityActions` becomes `CustomSemanticsAction` in the framework. Its basically allows the developer to define more complex interactions like maybe drag and drop. Actions are OS defined actions like tap, scroll, etc